### ISSUE TITLE
chinese fields

### ISSUE THREAD

**Issue by githubgtl** [zjunlp/KnowPrompt#23]
I want to change  English roberta model to chinese roberta，the data processed module showed that use_bert is ture,so the dataset return batch that is analysed by 5 variables to express, I don't find any module to accept the batch

**Comment by githubgtl** [comment]
I change it returns 4 variables,the progress can run normally but loss = nan and f1=0

**Comment by githubgtl** [comment]
maybe the true label is compeletly different from predicted labels, loss is very big

**Comment by njcx-ai** [comment]
Thanks for your attention. Perhaps you can try starting with Chinese BERT and simultaneously check for any discrepancies between the logits and labels.

**Comment by githubgtl** [comment]
> Thanks for your attention. Perhaps you can try starting with Chinese BERT and simultaneously check for any discrepancies between the logits and labels.

I have change the model but the loss is still nan,should I change the relation in the dataset into english

**Comment by njcx-ai** [comment]
Change the relation in the dataset into English would result in a semantic mismatch. You can utilize the debug mode of an IDE to troubleshoot and identify the issue.

**Comment by githubgtl** [comment]
> Change the relation in the dataset into English would result in a semantic mismatch. You can utilize the debug mode of an IDE to troubleshoot and identify the issue.

thank you for your reply, I fix the question which loss is nan but the best is only 40%

**Comment by zxlzr** [comment]
Hi, if you are using only few-shot samples, you should try to run experiment multiple times to obtain the average results, and the hyperparameter and data selection is is also very sensitive to the performance.

You can also try the code here https://github.com/zjunlp/LREBench, in which we have already tried the Chinese datasets.

**Comment by githubgtl** [comment]
thanks, I just download the LRE projects and run it , I found almost every epoch is f1=0.00ckpt, does your data appear this performance?

**Comment by githubgtl** [comment]
by the way ,the loss gets smaller and smaller,but eval_f1 is 0.00

**Comment by xxupiano** [comment]
Hello, what's the dataset and the dataset size you used? Maybe the dataset size is such small that the model can not study them well.

**Comment by githubgtl** [comment]
I use the same dataset as your readme refers

**Comment by githubgtl** [comment]
but train dataset is 6000 picked by your dataset

**Comment by xxupiano** [comment]
Can you provide which dataset you used and which script you ran?

**Comment by zxlzr** [comment]
Have you solved your issues?

**Comment by githubgtl** [comment]
> Have you solved your issues?

sorry , I just see this email, I solve it , thank you very much
